packages/importer/src/index.ts

```diff
@@ -135,23 +109,36 @@ class OneSchemaImporter extends EventEmitter {
     parent.append(this.iframe)
   }
 
-  launch(config: OneSchemaImporterConfig) {
+  /**
+   * Launch will show the OneSchema window and initialize the importer session
+   * @param launchConfig optionally pass in config overrides or values not passed into constructor
+   */
+  launch(launchConfig?: Partial<OneSchemaLaunchConfig>) {
     window.addEventListener("message", this.#eventListener)
     this.#show()
 
-    const { options, ...rest } = config
+    const mergedConfig = merge({}, this.#config, launchConfig)
+    const message: any = { messageType: "init" }
+    message.options = mergedConfig.config
```

Comment:
nb: i changed this to `config` in how it's presented because it feels like that's how we see it.. growing into a much larger thing, but now there's a disjoint with how it's passed in the init message. config is VERY overloaded in how i use it now. Should I leave this as options?